components/eamxx/cime_config/namelist_defaults_eamxx.xml

```diff
@@ -193,6 +193,10 @@ be lost if SCREAM_HACK_XML is not enabled.
       <moisture>moist</moisture>
       <!-- Frequency in physics steps to output a global hash over the dycore's
            in-fields. <= 0 disables hashing. -->
+      <enable_column_conservation_checks>false</enable_column_conservation_checks>
+      <enable_energy_fixer>true</enable_energy_fixer>
```

Comment:
Would you mind adding some metadata to these new XML entries? Namely, the "type" and "doc" metadata. We use these metadata entries when we auto-generate documentation for model inputs (which gets deployed on the [online docs](https://docs.e3sm.org/E3SM/EAMxx/user/eamxx_params/)).

Reply:
done, thanks